src/cljs/commiteth/handlers.cljs

```diff
@@ -320,7 +319,7 @@
     :http {:method     POST
            :url        "/api/user/address"
            :on-success #(do
-                          (dispatch [:assoc-in [:user [:address] address]])
+                          (dispatch [:assoc-in [:user :address] address])
```

Comment:
It was a bug here, when the wrong key was updated so nothing was shown in UI.

Reply:
Good catch